examples/build.gradle.kts

```diff
@@ -8,18 +8,11 @@ plugins {
 }
 
 allOpen.annotation("org.openjdk.jmh.annotations.State")
-
-repositories {
-    maven("https://dl.bintray.com/mipt-npm/kscience")
```

Comment:
See the comment above. I think it makes sense to leave them here for people to see which repositories to use.